release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -58,6 +58,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-installation-and-upgrade"]
 === Installation and upgrade
 
+[id=ocp-4.9-azure-cidr]
+==== Increased size of Azure subnets within the machine CIDR
+The {product-title} installation program for Microsoft Azure now creates subnets as large as possible within the machine CIDR. This lets the cluster use a machine CIDR appropriately sized to accommodate the number of nodes in the cluster.
```

Comment:
s/appropriately/that is appropriately ??
Might make the sentence a little more clear. I stumbled over that phrase the first time.